src/helpers/employeeService.ts

```diff
@@ -14,22 +14,26 @@ export const deleteEmployee = async (employeeId: number) => {
 	});
 };
 
-export const createEmployee = async (req: CreateEmployeeInputType) => {
+export const createEmployee = async (input: CreateEmployeeInputType) => {
```

Comment:
@iskandar-chacrasoftware The previous comment changes has been done here. I have moved the file to the separate folder.